#ifndef STANDARD_PARAMETER_NAME_H
#define STANDARD_PARAMETER_NAME_H

#define   ROSPARAM_NAME_STEP_SIZE_X          "step_size_x"
#define   ROSPARAM_NAME_STEP_SIZE_Y          "step_size_y"
#define   ROSPARAM_NAME_MAX_X                "max_x"
#define   ROSPARAM_NAME_MIN_X                "min_x"
#define   ROSPARAM_NAME_START_X              "start_x"
#define   ROSPARAM_NAME_MAX_Y                "max_y"
#define   ROSPARAM_NAME_MIN_Y                "min_y"
#define   ROSPARAM_NAME_START_Y              "start_y"
#define   ROSPARAM_NAME_GROUP_NAME           "group_name"
#define   ROSPARAM_NAME_STARTPOSE_JOINTNAMES "start_pose_joint_names"
#define   ROSPARAM_NAME_STARTPOSE_JOINTSTATE "start_pose_joint_states"
#define   ROSPARAM_NAME_TABLE_HEIGHT         "table_layer_height"
#define   ROSPARAM_NAME_OBSTACLE_HEIGHT      "obstacle_layer_height"
#define   ROSPARAM_NAME_LAYER_COUNT          "layer_count"
#define   ROSPARAM_NAME_LAYER_HEIGHT         "layer_height"

#endif
